contracts/mocks/ERC20FlashMintMock.sol

```diff
@@ -13,4 +16,29 @@ contract ERC20FlashMintMock is ERC20FlashMint {
     ) ERC20(name, symbol) {
         _mint(initialAccount, initialBalance);
     }
+
+    function mint(address account, uint256 amount) public {
+        _mint(account, amount);
+    }
+
+    function setFlashFee(uint256 amount) public {
+        _flashFeeAmount = amount;
+    }
+
+    function flashFee(address token, uint256 amount) public view virtual override returns (uint256) {
+        super.flashFee(token, amount);
```

Comment:
This super call is not needed and can be confusing, even if it is just a mock.

Reply:
The purpose of this super call is to cover the validation defined in the super implementation. I have opened another ticket to fix the flashFee function. #3331